pkg/preflights/host-preflight.yaml

```diff
@@ -47,6 +47,18 @@ spec:
         command: 'sh'
         args: ['-c', 'command -v umount']
     - hostOS: {}
+    - http:
+        collectorName: http-replicated-app
```

Comment:
I have a question here
does the preflights runner inherit proxy settings from the user's shell environment?

Reply:
So i call setProxyEnv for each command that executes the preflights and they way exec.Command works it will inherit the environment from the parent golang process. I wonder if i should modify that function to set the env vars to empty strings if nil or unset?

https://github.com/replicatedhq/embedded-cluster/pull/968/files#diff-0d2512143545f44ccab24560b90d4413dc15a3dd9e0d1723dc841ce2e94b3554R67-R80

Reply:
setting the env vars to empty strings if no proxy flags are passed makes perfect sense to me